config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,14 @@
+class Rack::Attack
+  # Allows 120 requests/IP in ~1 minutes
+  #        240 requests/IP in ~8 minutes
+  #        480 requests/IP in ~1 hour
+  #        960 requests/IP in ~8 hours (~2,880 requests/day)
+  (2..5).each do |level|
```

Comment:
@Pralish please provide tuning via a single variable that has a reasonable default value if not set